### PR TITLE
(MODULES-8303) improve IPv6 support and xinetd configuration

### PR DESCRIPTION
**Pull request by schwicke**
With this patch it is possible to configure rsyncd in dual-stack operation, listening to both IPv4 and IPv6.

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppetlabs-rsync",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (traylenator) [inline metadata.json]: Don't increase the version in the patch for the new feature. All the release bumps are automated.

File: manifests/server.pp [hunk 2]
```diff
@@ -39,22 +43,79 @@
...
   if $use_xinetd {
-    include xinetd
-    xinetd::service { 'rsync':
-      bind        => $address,
-      port        => '873',
-      server      => '/usr/bin/rsync',
-      server_args => "--daemon --config ${conf_file}",
-      require     => Package['rsync'],
+    include ::xinetd
+    if $enable_ipv4 {
```
Comment (traylenator) [inline manifests/server.pp]: This does not follow the style guide https://puppet.com/docs/puppet/5.5/style_guide.html#simple-resource-declarations

File: templates/module.erb [hunk 3]
```diff
@@ -45,9 +45,6 @@ hosts allow        = <%= Array(@hosts_allow).join(' ')%>
 <% if @hosts_deny -%>
 hosts deny         = <%= Array(@hosts_deny).join(' ')%>
 <% end -%>
-<% if @exclude -%>
```
Comment (traylenator) [inline templates/module.erb]: Don't understand why this has been removed.

File: manifests/server.pp [hunk 4]
```diff
@@ -39,22 +43,79 @@
...
+        service_name => $servicename,
+        require      => Package['rsync'],
+      }
+    }
+    if $enable_ipv6 {
+      $conf_file_ipv6 = "${conf_file}_ipv6"
+      concat { $conf_file_ipv6: }
+      concat::fragment { 'rsyncd_ipv6_conf_header':
+        target  => $conf_file_ipv6,
+        content => template('rsync/header_ipv6.erb'),
```
Comment (traylenator) [inline manifests/server.pp]: New template the template does not seeem to be in the MR ?

**Comment by schwicke** [comment]
Closing this MR, see ticket MODULES-8303

**Pull request closed without merging**